server/views/pages/visit/summary.njk

```diff
@@ -47,35 +47,37 @@
     {% set cancelledVisitReason = '' %}
 
       {% if visit.visitStatus === "CANCELLED" %}
-
+      
         {% set status = visit.outcomeStatus.split("_") %}
-        {%- for visitNote in visit.visitNotes %}
-          {% if visitNote.type == "VISIT_OUTCOMES" %}
-          {% set cancelledVisitReason = visitNote.text %}
-            
-            {% set cancelledBy %}
-              {% if status[0] === 'ADMINISTRATIVE' %}
-                {{- 'This visit was cancelled due to an administrative error with the booking.' -}}
-              {% elseif status[0] === 'DETAILS' %}
-                {{- 'This visit was cancelled as the details changed after booking.' -}}
-              {% else %}
-                {{- 'This visit was cancelled by the ' + status[0] | lower + '.' -}}
-              {% endif %}
-            {% endset %}
-
-            {% set cancelledVisitBannerHtml %}
-              {{ govukWarningText({
-                classes: "govuk-!-margin-bottom-0 govuk-!-padding-left-4",
-                html: '<span data-test="visit-cancelled-type">' + cancelledBy + '</span>',
-                iconFallbackText: "Warning"
-              }) }}
-            {% endset %}
 
-            {{ mojBanner({
-              classes: "govuk-!-padding-left-4",
-              html: cancelledVisitBannerHtml
-            }) }}
+        {% set cancelledBy %}
+          {% if status[0] === 'ADMINISTRATIVE' %}
+            {{- 'This visit was cancelled due to an administrative error with the booking.' -}}
+          {% elseif status[0] === 'DETAILS' %}
+            {{- 'This visit was cancelled as the details changed after booking.' -}}
+          {% elseif status[0] === 'BOOKER' %}
+            {{- 'This visit was cancelled by the visitor.' -}}
```

Comment:
Shouldn't this be the booker?

Reply:
https://dsdmoj.atlassian.net/browse/VB-4787

No Ed asked specifically that is said visitor